src/argus_htmx/settings.py

```diff
@@ -56,3 +57,10 @@
 DAISYUI_THEMES = get_json_env("DAISYUI_THEMES", DEFAULT_THEMES, quiet=True)
 DEFAULT_THEME_OVERRIDE = {}
 TAILWIND_THEME_OVERRIDE = get_json_env("TAILWIND_THEME_OVERRIDE", DEFAULT_THEME_OVERRIDE, quiet=True)
+
+MESSAGE_TAGS = {
+    messages.INFO: "alert-info",
+    messages.SUCCESS: "alert-success",
+    messages.WARNING: "alert-warning",
+    messages.ERROR: "alert-error",
+}
```

Comment:
Have you checked that this doesn't break the admin?

Reply:
good point

Reply:
django admin seems fine, but could you perhaps verify?

Reply:
decided to stick with default tags